packages/base/TypographyOverflow/src/TypographyOverflow/TypographyOverflow.tsx

```diff
@@ -115,9 +106,10 @@ export const TypographyOverflow = (props: Props) => {
     <Typography
       {...rest}
       style={extendedStyle}
-      className={cx(
-        classes.wrapper,
-        isMultiline ? classes.multiLine : '!block !whitespace-nowrap',
+      noWrap={!isMultiline}
+      className={twMerge(
+        '[-webkit-box-orient:vertical] overflow-hidden text-ellipsis pr-[0.9px]',
+        isMultiline ? '[display:-webkit-box] break-words' : 'block',
```

Comment:
From the styles, I think we also need to force `white-space` to `initial` on multiline. This might be because in other components white space gets inherited 
```suggestion
        isMultiline ? '[display:-webkit-box] break-words !whitespace-[initial]' : 'block',
```

Reply:
but it is initial, not inherit, right 🤔, I will test it out